crypto/asn1/a_gentm.c

```diff
@@ -163,56 +223,49 @@ ASN1_GENERALIZEDTIME *ASN1_GENERALIZEDTIME_set(ASN1_GENERALIZEDTIME *s,
     return ASN1_GENERALIZEDTIME_adj(s, t, 0, 0);
 }
 
+int ASN1_GENERALIZEDTIME_cmp_time_t(const ASN1_GENERALIZEDTIME *s, time_t t)
```

Comment:
ASN1_UTCTIME_cmp_time_t() already exists in master, this is for completeness of ASN1_TIME-types, especially since we are now moving from UTCTIME to GENERALIZEDTIME as expiration dates are entering the 2040s. It is commonly used to see if a certificate has expired. Although, I suppose the ASN1_TIME_cmp_time_t() makes more since, since it can be used against either type.